index.d.ts

```diff
@@ -1,10 +1,10 @@
 declare module 'react-native-geocoding' {
     export default class Geocoder {
-        init(apiKey: string, options: Object): void;
-        isInit(): boolean;
-        setApiKey(API_KEY: string): void;
-        from(...params: any[]): Promise<void>;
-        getFromLocation(address: string): Promise<any>;
-        getFromLatLng(lat: number, lng: number): Promise<any>;
+        static init(apiKey: string, options: Object): void;
```

Comment:
@snaquaye  shouldn't options be optional ? like in the docs
`static init(apiKey: string, options?: Object): void;`